### PR TITLE
Added accuracy property

### PR DESCRIPTION
**Pull request by benquan**
I have a added the accuracy property to be pulled to the project.

### CODE REVIEW

File: geo-location.html [hunk 1]
```diff
@@ -182,14 +195,16 @@
      *
      * @param {Position} pos A position object from the Geolocation API.
      */
-    _onPosition: function(pos) {
+    _onPosition: function (pos) {
```
Comment (ebidel) [inline geo-location.html]: keep as is

File: geo-location.html [hunk 2]
```diff
@@ -182,14 +195,16 @@
...
       this._setPosition(pos);
       this._setLatitude(pos.coords.latitude);
       this._setLongitude(pos.coords.longitude);
+      this._setAccuracy(pos.coords.accuracy)
```
Comment (ebidel) [inline geo-location.html]: end with `;`

File: geo-location.html [hunk 3]
```diff
@@ -199,11 +214,11 @@
      *
      * @param {Position} err The error that was returned.
      */
-    _onError: function(err) {
+    _onError: function (err) {
```
Comment (ebidel) [inline geo-location.html]: keep as is

File: geo-location.html [hunk 4]
```diff
@@ -199,11 +214,11 @@
...
       this.fire('geo-error', {
         error: err.code + ': ' + err.message,
         code: err.code
       });
     }
   });
-</script>
+</script>
```
Comment (ebidel) [inline geo-location.html]: newline

**Review by ebidel — CHANGES_REQUESTED** [review]

**Comment by benquan** [comment]
Changes ready

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (ebidel) [inline .vscode/settings.json]: let's remove this file.

**Review by ebidel — CHANGES_REQUESTED** [review]

**Comment by FluorescentHallucinogen** [comment]
@ebidel @benquan Do we need a separate property for this, while we can access it via the `position` object property?

```html
<geo-location
    watch-pos
    high-accuracy
    position="{{position}}">
</geo-location>

<p>Accuracy: [[position.coords.accuracy]]</p>
```

If yes, I can create a pull request that adds separate properties for all `position` object properties such as accuracy, altitude, altitude accuracy, heading, speed, timestamp. :wink:

**Comment by ebidel** [comment]
@FluorescentHallucinogen that's a good point. I exposed lat/long b/c those are the most common but you can get accuracy through `position`. 

I don't think we need properties for all the sub properties :)

**Comment by FluorescentHallucinogen** [comment]
@ebidel On the other hand, the separate properties are more easy to use. As you can see, not all users know / understand that accuracy, altitude, altitude accuracy, heading, speed, timestamp, etc. are hidden behind the `position` object.

**Comment by ebidel** [comment]
Every property does come at a cost (observers, listeners, etc.) I think examples and docs help with showing what's possible.

**Comment by FluorescentHallucinogen** [comment]
@ebidel Could this pull request be closed?

**Comment by ebidel** [comment]
Closing since `position` already gives you this info. Appreciate the PR @benquan!

**Pull request closed without merging**